### PR TITLE
Zarytski

### PR DESCRIPTION
**Pull request by QAEvgenyZarytsky**
add several easy tests... please do not judge strictly!

### CODE REVIEW

File: RubberDucksTest/src/test/java/mavenizer/staticPO/TablesPage.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.openqa.selenium.WebDriver;
+
+public class TablesPage {
+    CartPageTable example1;
+    CartPageTable example2;
+
+    WebDriver driver;
+    public TablesPage(WebDriver driver) {
+        this.driver = driver;
+        example1 = new CartPageTable (driver.findElement(By.xpath("//div[@id='box-checkout-summary']//tbody")));
```
Comment (andrej-khatkevich) [inline RubberDucksTest/src/test/java/mavenizer/staticPO/TablesPage.java:13]: Resources can be used here

File: RubberDucksTest/src/test/java/mavenizer/staticPO/ZarytskiTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+
+public class ZarytskiTest extends TestBase {
```
Comment (andrej-khatkevich) [inline RubberDucksTest/src/test/java/mavenizer/staticPO/ZarytskiTest.java:15]: We must place all tests for our team in BinTest.java (imho)

**Review by andrej-khatkevich — CHANGES_REQUESTED** [review]

File: RubberDucksTest/src/test/java/mavenizer/staticPO/ZarytskiTest.java (hunk 2 above)
Comment (QAEvgenyZarytsky) [reply RubberDucksTest/src/test/java/mavenizer/staticPO/ZarytskiTest.java:15]: Ok, I'll move my test to BinTest! thx

File: RubberDucksTest/src/test/java/mavenizer/staticPO/TablesPage.java (hunk 1 above)
Comment (QAEvgenyZarytsky) [reply RubberDucksTest/src/test/java/mavenizer/staticPO/TablesPage.java:13]: Thanks for the advice, I'll change!

Comment (andrej-khatkevich) [reply RubberDucksTest/src/test/java/mavenizer/staticPO/TablesPage.java:13]: Super! Thanks. See you on chat later

**Pull request closed without merging**